### PR TITLE
add azure_rm_keyvault_info

### PR DESCRIPTION
**Pull request by yungezz**
##### SUMMARY

add azure_rm_keyvault_info

##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@zikalino Please help review this PR when you're available! Thanks!

**Comment by Fred-sun** [comment]
@zikalino

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+                    type: complex
+                    returned: always
+                    contains:
+                        keys:
+                            description:
+                                Permissions to keys.
+                            type: list
+                            returned: always
+                            sample:
+                                - Get
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py]: maybe should be converted to lower case just to keep the same as main module

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+  - name: List Key Vault in current subscription
+    azure_rm_keyvault_info:
+'''
+
+RETURN = '''
+keyvaults:
+    description: List of Azure Key Vaults.
+    returned: always
+    type: complex
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py]: I think type should be list?

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_keyvault_info
+version_added: "2.9"
+short_description: Get Azure Key Vault facts.
+description:
+    - Get facts of Azure Key Vault.
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py:22]: I think we should have "Get info" considering renaming ;-) but that's minow

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+options:
+    resource_group:
+        description:
+            - The name of the resource Group to which the key vault belongs.
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py]: Group should be lower case

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py [hunk 5]
```diff
@@ -0,0 +1,323 @@
...
+
+'''
+
+EXAMPLES = '''
+  - name: Get Key Vault by name
+    azure_rm_keyvault_info:
+      resource_group: myResourceGroup
+      name: myVault
+
+  - name: List Key Vault in specific resource group
```
Comment (zikalino) [inline lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py]: key vaults

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py (hunk 3 above)
Comment (yungezz) [reply lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py:22]: i ever think about it. but i think `facts` here is more common used in ansible, so keep facts.

**Review by zikalino — APPROVED** [review]
Looks very good now. Just noticed a few really minor things, so it can be merged.

File: lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py (hunk 3 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_keyvault_info.py:22]: it's ok, such things we can rename later anyway if we decide to

**Comment by yungezz** [comment]
ready_for_review

**Pull request merged**